packages/rum-core/src/performance-monitoring/performance-monitoring.js

```diff
@@ -210,7 +210,7 @@ export default class PerformanceMonitoring {
          * related to this interaction.
          */
         const tr = transactionService.startTransaction(
-          `Click >> ${tagName}${additionalInfo}`,
+          `Click - ${tagName}${additionalInfo}`,
```

Comment:
Better character would be `:` instead of `-` as tagName can also contain `-` if it's a custom element.

Reply:
Also no space between the click and colon. Name would be `Click: <otherinfo>`

Reply:
I would prefer to limit the number of different characters we use in the name to avoid this type of issues. Unless there's a technical reason not to use `-`, specially since there are spaces to indicate the separation.

Reply:
Yes, there is indeed a technical reason and it is `Custom Elements`. You can refer the spec here https://html.spec.whatwg.org/multipage/custom-elements.html

Most of the custom elements would be written in the form `<customized-button>` and our click transaction would be `Click - CUSTOMIZED-BUTTON`. This feels bad from the naming perspective.

Reply:
I meant a technical reason in the sense that the character would not be shown correctly in the UI, which I don't think is the case here.  IMO, `Click - CUSTOMIZED-BUTTON`  is clear since there're space to indicate the separation, we similarly use spaces in xhr transaction name to separate the method from the url.

Reply:
Alternatively we can come up with a standard naming convention to use for all transaction and span names, but that's out of the scope of this PR.

Reply:
Created an issue: https://github.com/elastic/apm-agent-rum-js/issues/692